src/Sylius/Behat/Element/Admin/TopBarElement.php

```diff
@@ -24,7 +24,9 @@ public function hasAvatarInMainBar(string $avatarPath): bool
 
     public function hasDefaultAvatarInMainBar(): bool
     {
-        return strpos($this->getAvatarImagePath(), '/assets/admin/img/50x50.png') !== false;
+        $avatarElement = $this->getDocument()->find('css', 'i.ui.avatar.user.icon');
```

Comment:
Shouldn't this selector look more like:
```suggestion
        $avatarElement = $this->getDocument()->find('css', 'img.ui.avatar.image');
```
https://www.w3schools.com/cssref/sel_element_class.asp

Reply:
avatar is empty so we are checking `<i class="ui avatar icon user"></i>`, isn't that correct?

Reply:
Of course, when I wrote this comment the build were failing due to this line. The element was an image instead of an icon. I can't resolve this comment 😅